src/main/kotlin/app/ui/profilePreview/View/ProfilePreview.kt

```diff
@@ -0,0 +1,208 @@
+                            style {
+
+                                backgroundColor += Color.WHITE
+                                cursor = Cursor.HAND
+                                minWidth = 75.0.px
+                                minHeight = 75.0.px
+                                fontSize = 2.em
+                                textFill = c("#CC4141")
+                            }
+                            action {
+                                viewModel.listenedAudio(false)
+                                find(ProfilePreview::class).replaceWith(UserCreation::class)
+
+                            }
+                        }
+                    }
+
+                    label("REDO") {
+                        hide()
+                        audioListened.subscribeBy(
+                                onNext = { if (it) show() else hide() }
+                        )
+                    }
+                }
+
+                stackpane {
+                    circle {
+
+                        style {
+                            radius = 120.0
+                            fill = c("#E5E5E5")
+                        }
+
+                    }
+                    iconHash.subscribeBy(
+                            onNext = {
+                                add(NewUserButton)
+                                NewUserButton.svgHash = it          // update iconhash from subject
+                            }
+                    )
+
+
+                    NewUserButton.profIcon.action {
+                        viewModel.listenedAudio(true)
+                    }
+                }
+
+
+                vbox {
+                    spacing = 12.0
+                    alignment = Pos.CENTER
+                    rightArrow.fill = c("#FFFFFF")
+                    stackpane {
+                        hide()
+                        audioListened.subscribeBy(                           // check if the audio has been listened if so, display the button NEXT if not just hide it
+                                onNext = { if (it) show() else hide() }
+                        )
+                        circle {
+
+                            style {
+                                radius = 55.0
+                                fill = c("#E5E5E5")
+                            }
+
+                        }
+                        button("", rightArrow) {
+
+
+                            importStylesheet(ButtonStyles::class)
+                            addClass(ButtonStyles.roundButton)
+                            style {
+                                backgroundColor += c("#CC4141")
+                                cursor = Cursor.HAND
+                                minWidth = 75.0.px
+                                minHeight = 75.0.px
+                                fontSize = 2.em
+                                textFill = c("#CC4141")
+                            }
+
+                            action {
```

Comment:
don't have an action if there's nothing to put here?

Reply:
i believe it is there to 'doSomething' when the next button is clicked, but we don't have that screen yet. so i'll remove for now